src/core/vr-scene.js

```diff
@@ -7,6 +7,11 @@
         value: function() {
           this.attachEventListeners();
           this.setupScene();
+
+          // handle fullscreen changes
+          document.addEventListener('fullscreenchange', this.fullscreenChange.bind(this), false);
+          document.addEventListener('mozfullscreenchange', this.fullscreenChange.bind(this), false);
```

Comment:
though it'll probably never happen, if Firefox starts supporting the non-prefixed `'fullscreenchange'`, this will get called twice